binary_database_files/models.py

```diff
@@ -4,82 +4,75 @@
 
 import six
 
-from . import settings as _settings
-
 from django.conf import settings
 from django.db import models
 from django.utils import timezone
 
-from database_files import utils
-from database_files.utils import write_file, is_fresh
-from database_files.manager import FileManager
+from django.db.models import BinaryField
+
+from binary_database_files import utils
+from binary_database_files.utils import write_file, is_fresh
+from binary_database_files.manager import FileManager
+
+from . import settings as _settings
 
 class File(models.Model):
-    
+
     objects = FileManager()
-    
+
     name = models.CharField(
         max_length=255,
         unique=True,
         blank=False,
         null=False,
         db_index=True)
-    
+
     size = models.PositiveIntegerField(
         db_index=True,
         blank=False,
         null=False)
 
-    _content = models.TextField(db_column='content')
-    
+    content = BinaryField(
+        blank=False,
+        null=False)
+
     created_datetime = models.DateTimeField(
         db_index=True,
         default=timezone.now,
         verbose_name="Created datetime")
-    
+
     _content_hash = models.CharField(
         db_column='content_hash',
         db_index=True,
         max_length=128,
         blank=True, null=True)
-    
+
     class Meta:
-        db_table = 'database_files_file'
-    
+        db_table = 'binary_database_files_file'
```

Comment:
I have come round to the conclusion that we will keep the new name to avoid confusion, and we will write docs to explain how to migrate existing data.